build.xml

```diff
@@ -459,34 +513,36 @@
         ]]>
     </scriptdef>
 
-    <scriptdef name="foreach-js" language="javascript">
-        <attribute name="target"/>
-        <attribute name="property"/>
-        <![CDATA[
-            var input = project.getProperty(attributes.get("property"));
-            var items = input.split(/\s*,\s*/);
-            for (i=0; i < items.length; i++) {
-              // create and use a Task via Ant API
-              antc = project.createTask("antcall");
-              var prop = antc.createParam();
-              prop.setName("item");
-              prop.setValue(items[i]);
-
-              antc.setTarget(attributes.get("target"));
-              antc.perform();
-            }
-        ]]>
-    </scriptdef>
-
     <target name="git.clone">
-        <echo message="Cloning ${item}..."/>
-        <exec executable="${git}" outputproperty="git.output" dir="${repos}">
+        <echo message="Cloning ${origin}..."/>
+        <exec executable="${git}" outputproperty="git.output" dir="${repos}" failonerror="true">
```

Comment:
Is this newly added attribute required? In my testing, it prevents us from re-running "1. Clone all repositories & resources." Our maintenance steps depend on the ability to re-run this, to fetch new maven dependencies and even possibly clone newly added repositories. The previous error condition was totally harmless. It returns the following for each repository that already exists:

```
git.clone:
     [echo] Cloning https://github.com/HistoryAtState/hsg-shell.git...
     [exec] Result: 128
     [echo] fatal: destination path 'hsg-shell' already exists and is not an empty directory.
```

I'm open to other solutions, but this change breaks our system.

Reply:
Fixed with https://github.com/evolvedbinary/hsg-project/pull/3/commits/d10ec0134b3b7247f9fdd9ce9a58518bc390bd50:
I have set a dynamic `failonerror` attribute value, achieved by pre-checking if the repo is existing or not. 
The build will now only fail on error, if the respective repo is not yet existing. If it is existing and an error occurs, the build will finish and display useful error messages.

Reply:
@plutonik-a Thank you! I see the updated steps:

```text
check-if-repo-exists:
     [echo] The repo 'hsg-shell' already exists and cannot be overwritten. Please delete this repo and try again.
```

I think the 2nd sentence could be softened to something like:

> If you need to re-clone this repo, please delete 'hsg-shell' and try again.